### PR TITLE
optimize Code

### PR DESCRIPTION
**Pull request by talha828**
optimize your code a little bit further especially the experience portion user can add multiple experiences

### CODE REVIEW

**Comment by talha828** [comment]
here are some sample images 
WhatsApp Image 2023-05-10 at 18 54 44 (1)
WhatsApp Image 2023-05-10 at 18 54 44

File: lib/src/components/section_bottom_buttons.dart [hunk 1]
```diff
@@ -49,3 +49,4 @@ class _AnimateButtonState extends State<AnimateButton> {
     );
   }
 }
+
```
Comment (JordyHers) [inline lib/src/components/section_bottom_buttons.dart:52]: Here can you set a blank line

File: lib/src/layout/layout_modern.dart [hunk 2]
```diff
@@ -338,116 +338,69 @@ class _LayoutModernState extends State<LayoutModern> {
                                       ),
                                     ),
                                     Config.spaceBox(Config.smallSpacer),
-                                    AnimatedShakingBuilder(
-                                      autoPlay: isDragged,
-                                      child: SizedBox(
-                                        width: widget.w,
-                                        child: Row(
-                                          mainAxisAlignment:
+                                    for(int i=0;i<widget.data.experience.length;i++)
```
Comment (JordyHers) [inline lib/src/layout/layout_modern.dart:341]: can you please explain me that approach ?

File: lib/src/utils/strings.dart [hunk 3]
```diff
@@ -16,11 +16,32 @@ class Str {
       email: 'matthewsmith@join.com',
       phoneNumber: '+ 1 (246)869 453 00',
       bio: Str.bioText,
-      experienceTitle: 'Creative Director',
-      experiencePlace: 'at Uber',
-      experiencePeriod: 'Sep 2018 - Jan 2020',
-      experienceLocation: 'New york City',
-      experienceDescription: Str.myRoleText,
+      experience: [
```
Comment (JordyHers) [inline lib/src/utils/strings.dart:19]: Great idea @talha828 thanks ✅

**Review by JordyHers — CHANGES_REQUESTED** [review]

**Comment by JordyHers** [comment]
@talha828  Can you also export the ExperienceData model in the `fluter_resume_template` file.

File: lib/src/components/section_bottom_buttons.dart (hunk 1 above)
Comment (talha828) [reply lib/src/components/section_bottom_buttons.dart:52]: I don`t noticed when I was pulling

File: lib/src/layout/layout_modern.dart (hunk 2 above)
Comment (talha828) [reply lib/src/layout/layout_modern.dart:341]: Here we can use a list view as well but this is very easy and efficient and clean

File: lib/src/utils/strings.dart (hunk 3 above)
Comment (talha828) [reply lib/src/utils/strings.dart:19]: when I was working on your project I was pretty impressed with your coding style. it`s is a very well define and proper code

**Comment by talha828** [comment]
review done

**Pull request merged**